dt-core/admin/menu/tabs/admin-endpoints.php

```diff
@@ -147,6 +156,57 @@ public function update_custom_field_translations( WP_REST_Request $request ){
         }
     }
 
+    public function data_clean_up( WP_REST_Request $request ) {
+        $params = $request->get_params();
+        if ( isset( $params['post_type'] ) ) {
+            $post_type = $params['post_type'];
+
+            // Build list of existing field ids.
+            $field_settings = DT_Posts::get_post_field_settings( $post_type, false );
+            $field_ids = array_keys( $field_settings );
+
+            // Identify deleted field activity logs.
+            global $wpdb;
+
+            $id_sql = [];
+            foreach ( $field_ids as $id ) {
+                $id_sql[] = "(log.object_subtype NOT LIKE '" . $id . "%')";
```

Comment:
@kodinkat what is the `%` wildcard at the end here for?

Reply:
@corsacca to accommodate `communication_channel` field type naming conventions.

![Screenshot 2023-10-20 at 11 00 25](https://github.com/DiscipleTools/disciple-tools-theme/assets/19330800/bc9a14cf-3f1d-4081-bf4e-324e7dcacf85)